src/FOM/UserBundle/Resources/public/css/userlist.datatables.css

```diff
@@ -0,0 +1,86 @@
+.dataTables_filter label input {
```

Comment:
@debugteam, 
1. I will not open an issue for that
2. Stop to use `!important` declaration
3. No global selectors like `.tdsmall`